### ISSUE TITLE
Slider centered scaling (alt) does not always match expectations

### ISSUE THREAD

**Issue by hiderikzki** [ppy/osu#31154]
### Type

Game behaviour

### Bug description

When scaling positions in editor with multiple sliders while holding alt, to keep the center of elements the same, it does not always work as such, and instead depends on the slider orientation (?), and the distance difference on the x- or y-axis depending on the slider formation.

Sometimes shown as bias towards scaling to one side more than the other, or scaling very little and moving sliders way too much.

Expectations would be for it to work the same as scaling circles (as shown first in the video), with no issues to any direction

### Screenshots or videos

https://github.com/user-attachments/assets/d58b0c9f-6340-404a-9215-c70275f3f7e1

### Version

2024.1208.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/18149922/compressed-logs.zip)

**Comment by bdach** [comment]
To confirm intent, you would be expecting the resize operation to change the shape of the slider itself?

**Comment by hiderikzki** [comment]
> To confirm intent, you would be expecting the resize operation to change the shape of the slider itself?

no, i expect it to move equal distance to the scaled direction, example in this video, scaling horizontally seems to work as expected, not moving another side more than the other, but scaling vertically moves the other side drastically more.

https://github.com/user-attachments/assets/f17b1d66-418d-45df-8102-18059e35c875
